### PR TITLE
Multilevel Support (updated)

### PR DESCRIPTION
**Pull request by transcranial**
https://github.com/mcollina/levelgraph/pull/60 is outdated. This pull request allows for multilevel compatibility with interval changes in levelgraph and multilevel APIs:

+ `db.db.approximateSize` not exposed by `multilevel.client()` - check for `db.db`
+ allow for triples putStream by activating `level-ws`, given `db.createWriteStream` is deprecated in levelup - needs `leveldb.isClient` check for multilevel
+ `patchQuery` taken from above pull request

### CODE REVIEW

File: lib/levelgraph.js [hunk 1]
```diff
@@ -110,6 +110,22 @@ module.exports = function levelgraph(leveldb, options) {
...
+  
+  query.forEach(function(e) {
+    utilities.defs.spo.forEach(function(key) {
+      if (typeof e[key] === 'object' &&
+          e[key].constructor !== Variable) {
+        e[key] = new Variable(e[key].name);
+      }
+    });
+  });
+}
```
Comment (mcollina) [inline lib/levelgraph.js]: why this is needed?

**Comment by mcollina** [comment]
Thanks so much! Would you like to help maintaining LevelGraph?

**Comment by transcranial** [comment]
@mcollina Happy to help! Good point on patchQuery - it was from the old pull request. Fixed test to use `db.v`.

**Comment by mcollina** [comment]
I've added you to the committers, add yourself to the README and package.json, and I'll merge this!

**Comment by transcranial** [comment]
Done.

**Pull request merged**